src/main/webapp/WEB-INF/templates/views/privileged/my_flat.html

```diff
@@ -4,35 +4,123 @@
+                        <div class="regpaddingbottom padding-top-10">
+                            <button style="display: none" id="toggle-button-addition-show" class="roundedbutton min-width-300" onclick="showVisibilityOfAdditionElement()">Show option of Controller Unition Creation</button>
+                        </div>
+                        <div class="regpaddingbottom padding-top-10">
+                            <button style="display: block" id="toggle-button-addition-hide" class="roundedbutton min-width-300" onclick="hideVisibilityOfAdditionElement()">Hide option of Controller Unition Creation</button>
+                        </div>
+                    </div>
+                    <!-- samotny formular co sa skryva -->
+                    <div class="grid col-sm-12 " id="addition-toggle">
+                        <div class="row justify-content-center regpadding rowinrightcol createuser">
+                            <h1 class="reglogheader">Create new Controller Unit for this flat</h1>
+                        </div>
+                        <div class="grid regpadding">
+                            <div class="row justify-content-center">
+                                <div class="regpaddingtopbottom padding-right-5">
+                                    <label for="controller-uid">UID:</label>
+                                    <div><input type="text" id="controller-uid" placeholder="should be generated" required></div>
+                                </div>
+                                <div class="regpaddingtopbottom padding-left-5">
+                                    <label for="controller-dip_address">DIP Address:</label>
+                                    <div><input type="text" id="controller-dip_address" placeholder="1-255" required></div>
+                                </div>
+                            </div>
+                            <div class="row justify-content-center">
+                                <div class="regpaddingtopbottom">
+                                    <label for="controller-zwave">Zwave identifier:</label>
+                                    <div><input class="min-width-300" type="text" id="controller-zwave" placeholder="ZW-XXX" required></div>
+                                </div>
+                            </div>
+                            <!-- HIDE THIS BUT KEEP THIS ITS IMPORTANT-->
+                            <div class="grid regpadding" hidden>
+                                <div class="row justify-content-center rowinrightcol">
+                                    <div class="regpaddingtopbottom">
+                                        <label for="controller-flat-id">FLAT ID</label>
+                                        <div><select class="min-width-300" id="controller-flat-id">
+                                            <option th:default="${flat.id}" th:value="${flat.id}" th:text="${'id je: ' + flat.id}"></option>
```

Comment:
databazove pk id na fronte asi ci? to je ta vladova vec co neznasa, ale nemas tu moc na vyber pravdepodobne, lebo sa tam nedal iny identifikator

Reply:
Hej je to napi.u, ale aspon je to skryté. Rozmýšľal som do 2ej rána jak to spraviť čo najlepšie.... Hlavne som tam nechcel mať nejaky obrovsky dropdown, je to take divne cele....